packages/@aws-cdk/aws-appmesh/test/test.gateway-route.ts

```diff
@@ -0,0 +1,195 @@
+import * as cdk from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+
+import * as appmesh from '../lib';
+
+export = {
+  'When creating a GatewayRoute': {
+    'should have expected defaults'(test: Test) {
+      // GIVEN
+      const stack = new cdk.Stack();
+
+      // WHEN
+      const mesh = new appmesh.Mesh(stack, 'mesh', {
+        meshName: 'test-mesh',
+      });
+
+      const virtualGateway = new appmesh.VirtualGateway(stack, 'gateway-1', {
+        listeners: [appmesh.VirtualGatewayListener.httpGatewayListener()],
+        mesh: mesh,
+      });
+
+      const virtualService = new appmesh.VirtualService(stack, 'vs-1', {
+        mesh: mesh,
+      });
+
+      // Add an HTTP Route
+      virtualGateway.addGatewayRoute('gateway-route', {
+        routeSpec: appmesh.GatewayRouteSpec.httpRouteSpec({
+          routeTarget: virtualService,
+        }),
+        gatewayRouteName: 'gateway-route',
+      });
+
+      // THEN
+      expect(stack).to(
+        haveResourceLike('AWS::AppMesh::GatewayRoute', {
+          GatewayRouteName: 'gateway-route',
+          Spec: {
+            HttpRoute: {
+              Action: {
+                Target: {
+                  VirtualService: {
+                    VirtualServiceName: {
+                      'Fn::GetAtt': ['vs1732C2645', 'VirtualServiceName'],
+                    },
+                  },
+                },
+              },
+              Match: {
+                Prefix: '/',
+              },
+            },
+          },
+        }),
+      );
+      test.done();
+    },
+    'should be able to add multiple routes'(test: Test) {
+      // GIVEN
+      const stack = new cdk.Stack();
+
+      // WHEN
+      const mesh = new appmesh.Mesh(stack, 'mesh', {
+        meshName: 'test-mesh',
+      });
+
+      const virtualGateway = new appmesh.VirtualGateway(stack, 'gateway-1', {
+        listeners: [appmesh.VirtualGatewayListener.httpGatewayListener()],
+        mesh: mesh,
+      });
+
+      const virtualService = new appmesh.VirtualService(stack, 'vs-1', {
+        mesh: mesh,
+      });
+      const virtualService2 = new appmesh.VirtualService(stack, 'vs-2', {
+        mesh: mesh,
+      });
+
+      virtualGateway.addGatewayRoutes([
+        {
+          id: 'gateway-route',
+          props: {
+            routeSpec: appmesh.GatewayRouteSpec.httpRouteSpec({
+              routeTarget: virtualService,
+            }),
+            gatewayRouteName: 'gateway-route',
+          },
+        },
+        {
+          id: 'gateway-route2',
+          props: {
+            routeSpec: appmesh.GatewayRouteSpec.httpRouteSpec({
+              routeTarget: virtualService2,
+              match: {
+                prefixPath: '/echo',
+              },
+            }),
+            gatewayRouteName: 'gateway-route2',
+          },
+        },
+      ]);
+
+      // THEN
+      expect(stack).to(
+        haveResourceLike('AWS::AppMesh::GatewayRoute', {
```

Comment:
Have this on one line please:

```suggestion
      expect(stack).to(haveResourceLike('AWS::AppMesh::GatewayRoute', {
```

This way, the indentation at the end will be much more natural.

Same comment everywhere in the tests.